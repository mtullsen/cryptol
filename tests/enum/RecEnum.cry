enum List = Nil | Cons Integer List
