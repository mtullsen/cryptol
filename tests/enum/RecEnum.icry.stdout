Loading module Cryptol
Loading module Cryptol
Loading module Main

[error] Invalid self dependency:
    â€¢ type Main::List, defined at 1:6--1:10
